src/rgw/driver/rados/rgw_sync_counters.cc

```diff
@@ -26,3 +27,35 @@ PerfCountersRef build(CephContext *cct, const std::string& name)
 }
 
 } // namespace sync_counters
+
+
+namespace sync_deltas {
+
+//std::string rgw_sync_delta_counters_key = "rgw_sync_delta";
+
+void add_rgw_sync_delta_counters(PerfCountersBuilder *lpcb) {
+  lpcb->set_prio_default(PerfCountersBuilder::PRIO_USEFUL);
+  lpcb->add_time(l_rgw_datalog_sync_delta, "sync_delta", "Sync Delta between data log shard");
```

Comment:
in what units are these times reported? seconds? ms? ns? this should be specified here